components/gitpod-protocol/package.json

```diff
@@ -57,6 +57,8 @@
         "exit": true
     },
     "dependencies": {
+        "@connectrpc/connect": "1.1.2",
+        "@gitpod/public-api": "0.1.5",
```

Comment:
Hm, this dependenciy feels a bit weird. What's the reason to not put the converter into the typescript lib exported by public-api? :thinking:

Reply:
You mean reverse dependency? I don't think public-api should pull gitpod-protocol.
We need some place where we share converter between dashboard and server.

Reply:
Yes, probably not the single package that we have there now that contains the connect types.
But maybe having a separate one next to it... not sure if it's worth it though. Also can't pinpoint why it feels weird to me. :upside_down_face: